addons/mrp/tests/test_order.py

```diff
@@ -3395,6 +3395,7 @@ def test_replan_mo_after_updating_bom(self):
         """
 
         self.bom_3.write({'product_uom_id': self.uom_unit.id})
+        self.workcenter_1.resource_calendar_id.tz = 'Asia/kolkata'
```

Comment:
I Saw it [my pr](https://github.com/odoo/odoo/commit/9a6f22ab918b8f4924e75c6e604a9be80b79fe5e#diff-d5ae7c2abfc76061226a2b4a56afebe695b0ff5de4b81d24be8fc7646cd0f041R3396)
Maybe
```suggestion
        self.env.company.resource_calendar_id.tz = 'Asia/kolkata'
```
seems less prone to setup changes